public/docs/_examples/server-communication/dart/lib/toh/hero_service.dart

```diff
@@ -35,7 +35,7 @@ class HeroService {
 
   // #docregion addhero, addhero-sig
```

Comment:
Yes, start and end can be a region list (it helps reduce the number of extra lines in the file ... and makes it a bit more readable).